latest.html

```diff
@@ -98,5 +98,8 @@ <h3>what's changed.</h3>
             <a href="https://www.linkedin.com/in/anson-yu-231336141/" target="_blank"><img src="images/people/anson.jpg" alt="anson yu"></a>
         </div>
     </div>
+
+    !--Link to JavaScript File-->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
ok wait jokes this page never had jquery in it so I don't think we need to add it. I might've mistakingly put it in the issue description, my bad